naff/models/discord/role.py

```diff
@@ -220,3 +220,32 @@ async def move(self, position: int, reason: str | Missing = MISSING) -> "Role":
             self._guild_id, [{"id": self.id, "position": position}], reason
         )
         return self
+
+    # START: fail if not all permissions are manageable
+    async def clone(self) -> "Role":
+        return await self.guild.create_role(**self.to_dict())
+
+    # END: fail if not all permissions are manageable
+
+    # START: only set manageable permissions
+    async def clone(self) -> "ClonedRole":
```

Comment:
Why have you redefined `clone`

Reply:
since no one answered on Discord I implemented both ways to resolve the issue

one definition will be removed